packages/core/src/utils/getSCMigrationRef/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './getSCMigrationRef'
```

Comment:
why not attach these to `utils/index.js` so you can use one destructured import for utils?

Reply:
I don't feel strongly about doing it this way, but at some point we might want to consider breaking the utils out into separate modules if that file gets too bloated. I'll move the new helper in there to stick to the current pattern.